common/src/test/java/org/conscrypt/javax/net/ssl/SSLContextTest.java

```diff
@@ -664,14 +665,21 @@ public void test_SSLContext_SSLv3Unsupported() throws Exception {
     }
 
     private static void assertContentsInOrder(List<String> expected, String... actual) {
+        List<String> actualList = Arrays.asList(actual);
         if (expected.size() != actual.length) {
             fail("Unexpected length. Expected len <" + expected.size() + ">, actual len <"
                     + actual.length + ">, expected <" + expected + ">, actual <"
-                    + Arrays.asList(actual) + ">");
+                    + actualList + ">");
         }
-        if (!expected.equals(Arrays.asList(actual))) {
-            fail("Unexpected element(s). Expected <" + expected + ">, actual <"
-                    + Arrays.asList(actual) + ">");
+
+        if (System.getProperty("os.name", "").toLowerCase().startsWith("windows")) {
```

Comment:
Very Optional:  We do have an `isWindows()` method (which does the same thing) but it's currently living in [`SSLSocketVersionCompatibilityTest`](https://github.com/google/conscrypt/blob/master/common/src/test/java/org/conscrypt/javax/net/ssl/SSLSocketVersionCompatibilityTest.java#L2083-L2098)... It could do with moving to [`TestUtils`](https://github.com/google/conscrypt/blob/master/testing/src/main/java/org/conscrypt/TestUtils.java) along with its friends so we could use it here :)